projects/plugins/jetpack/tests/php/modules/masterbar/test-class-admin-menu.php

```diff
@@ -561,36 +562,4 @@ public function hide_menu_based_on_submenu_provider() {
 			),
 		);
 	}
-
-	/**
-	 * Check if the dashboard switcher is registered correctly.
-	 */
-	public function test_register_dashboard_switcher() {
```

Comment:
I've moved this test in a separate test class since after the refactor it's now unrelated to admin-menu classes.